lab-anthony/greet-cli.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const getGreet = require(__dirname + '/lib/greet');
```

Comment:
I usualy name my variables for a module the same as a file name :)
it makes it easy to tell where things are comming from when things get complex